diff --git a/build.sbt b/build.sbt
<<<<<<< HEAD
index e80b2a5..b1989d9 100644
=======
index 2187fe12..2319fc95 100644
>>>>>>> ec1b075... Merge pull request #1054 from ucb-bar/use-published-deps
--- a/build.sbt
+++ b/build.sbt
@@ -162,7 +162,7 @@ lazy val testchipip = (project in file("generators/testchipip"))
 
 lazy val chipyard = (project in file("generators/chipyard"))
<<<<<<< HEAD
   .sourceDependency(testchipip, testchipipLib)
   .dependsOn(rocketchip, boom, hwacha, sifive_blocks, sifive_cache, utilities, iocell,
=======
   .dependsOn(testchipip, rocketchip, boom, hwacha, sifive_blocks, sifive_cache, iocell,
>>>>>>> ec1b075... Merge pull request #1054 from ucb-bar/use-published-deps
-    sha3, // On separate line to allow for cleaner tutorial-setup patches
+//    sha3, // On separate line to allow for cleaner tutorial-setup patches
     dsptools, `rocket-dsptools`,
     gemmini, icenet, tracegen, cva6, nvdla, sodor)
   .settings(libraryDependencies ++= rocketLibDeps.value)
<<<<<<< HEAD
@@ -227,11 +227,11 @@ lazy val sodor = (project in file("generators/riscv-sodor"))
=======
@@ -203,10 +203,10 @@ lazy val sodor = (project in file("generators/riscv-sodor"))
>>>>>>> ec1b075... Merge pull request #1054 from ucb-bar/use-published-deps
   .settings(libraryDependencies ++= rocketLibDeps.value)
   .settings(commonSettings)
 
-lazy val sha3 = (project in file("generators/sha3"))
-  .dependsOn(rocketchip, chisel_testers, midasTargetUtils)
-  .settings(libraryDependencies ++= rocketLibDeps.value)
-  .settings(commonSettings)
+//lazy val sha3 = (project in file("generators/sha3"))
+//  .dependsOn(rocketchip, chisel_testers, midasTargetUtils)
+//  .settings(libraryDependencies ++= rocketLibDeps.value)
+//  .settings(commonSettings)
 
 lazy val gemmini = (project in file("generators/gemmini"))
   .dependsOn(testchipip, rocketchip, chisel_testers)
